integration/tests/lookup_by_resourceid_test.go

```diff
@@ -70,5 +70,5 @@ func TestLookupByResourceID(t *testing.T) {
 			})
 	}
 
-	RawUrlFollowupTests(t, "/v1/cloud/resourceid/" + resId, tsDuring)
+	RawUrlFollowupTests(t, "/v1/cloud/resourceid/"+resId, tsDuring)
```

Comment:
info: I am guessing this is some kind of auto-formatting? is it preferred we don't put a space after the string when concatenating? If so, TIL I will try to be mindful of that.

Reply:
Goimports did this :)